src/database/models/severa.ts

```diff
@@ -1,9 +1,15 @@
 /**
  * What Flextime returns from Severa.
  */
-interface Flextime {
+export interface Flextime {
 	totalFlextimeBalance: number | null; 
   monthFlextimeBalance: number | null;
 }
 
-export default Flextime;
\ No newline at end of file
+export interface TotalTime {
+  expectedHours: number;
+  enteredHours: number;
+  quantity: number;
+  totalBillableTime: number;
+  nonBillableProject: number;
+}
```

Comment:
I believe that those should not be in database/models, but in types/severa.
Also, something is wrong with formatting?